tests/acompat.py

```diff
@@ -118,17 +118,5 @@ async def wait_timeout(self, timeout):
         await asyncio.wait_for(self.wait(), timeout)
 
 
-class Queue(queue.Queue):  # type: ignore[type-arg]  # can be dropped after Python 3.8
-    """
-    A Queue subclass with an interruptible get() method.
-    """
-
-    def get(self, block: bool = True, timeout: float | None = None) -> Any:
-        # Always specify a timeout to make the wait interruptible.
-        if timeout is None:
-            timeout = 24.0 * 60.0 * 60.0
-        return super().get(block=block, timeout=timeout)
-
-
-class AQueue(asyncio.Queue):  # type: ignore[type-arg]
-    pass
+Queue = queue.Queue
+AQueue = asyncio.Queue
```

Comment:
Should it be got `from psycopg.psycopg._acompat import AQueue, Queue` or just leaving it so?